Board.gd

```diff
@@ -16,8 +14,8 @@ func _ready():
 			$Tiles.add_child(tile)
 
 	$Pieces.add_child(self.piece)
-	var tile = self.get_tile(Vector2(4, 4))
-	tile.set_piece(self.piece)
+	var initialTile = self.get_tile(Vector2(4, 4))
+	initialTile.set_piece(self.piece)
```

Comment:
👍

Reply:
This is a matter of personal taste, I like to follow python styling in GDcripts (Because of the similarity). I would use snake_case for variable declarations and CamelCase for classes.
```suggestion
	var initial_tile = self.get_tile(Vector2(4, 4))
	initial_tile.set_piece(self.piece)
```